src/database/hooks/useSaveAnonChatHook.ts

```diff
@@ -28,8 +28,12 @@ const useSaveAnonChatHook = () => {
   };
 
   const helperGoToAnonymousChat = async (object: InitAnonymousChatData, from: string) => {
-    const channelListSchema = await helperFindChatById(object);
-    goToChatScreen(channelListSchema, from);
+    try {
+      const channelListSchema = await helperFindChatById(object);
+      goToChatScreen(channelListSchema, from);
+    } catch (e) {
+      console.log('error helperGoToAnonymousChat', e);
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling here is a good addition, but it would be better to handle the error in a more meaningful way rather than just logging it. Consider adding some user feedback or retry logic.



<!-- This is an auto-generated comment by OSS CodeRabbit -->